README.md

```diff
@@ -15,9 +15,7 @@ HashiCorp Terraform.
 
 ## Overview
 
-The CDK for Terraform project helps users define infrastructure resources using supported programming languages and generates a Terraform configuration in JSON. Users can then use the Terraform or the CDK for Terraform CLI to deploy the application.
-
-The project uses [AWS CDK](https://aws.amazon.com/cdk/) [constructs](https://github.com/aws/constructs) to define applications.
+CDK for Terraform apps are written in one of the supported programming languages and can leverage the entire ecosystem of Terraform [modules and providers](./docs/working-with-cdk-for-terraform/using-providers-and-modules.md). The application [synthesizes](./docs/working-with-cdk-for-terraform/synthesizing-config.md) Terraform configuration in JSON. Users can then use the Terraform or the CDK for Terraform CLI to deploy the application.
```

Comment:
Nice addition!

Small suggestion on the last sentence... `Users can use...`

How about:
```Terraform or the CDK for Terraform can then be used to deploy the application.```

Reply:
See https://github.com/hashicorp/terraform-cdk/pull/550/commits/743073c1f69589b4f3456a53bf3a7cce08add7b1